Begin.py/begin.py

```diff
@@ -6,8 +6,6 @@
 # Naam: Annemarleen Bosma
 #
 
-
-
 """
```

Comment:
ik heb PyCharm de whitelines voor me laten formatten. Python heeft formatting standaarden ie zeggen hoeveel whitelines waar horen, dit kan je in een IDE met 1 klik laten fixen. :)
PyCharm is gratis voor studenten trouwens.